.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: JSON Schema Validation
+
+on: [push]
```

Comment:
Let's do this on PRs as well. Then we know if yaml files are correct before merging.

Reply:
When a PR is opened you always have to push the commits, or am I missing something.

Reply:
only if you have access to this repo :wink: outside PRs don't involve pushes

Reply:
Oh, I didn't consider that.

Reply:
So, just add `pull_request` or something else?

Reply:
yep (here are all triggers explained: https://help.github.com/en/actions/reference/events-that-trigger-workflows)